tests/scenarios/text-widget.spec.js

```diff
@@ -0,0 +1,59 @@
+const _ = require('lodash');
+const server = require('../server');
+const steps = require('../steps');
+
+let contentMainBlockSelector;
+let richTextSelector;
+
+module.exports = Object.assign(
+  {
+    before: (client, done) => {
+      client.resizeWindow(1200, 800);
+
+      this._server = server.create();
+      this._server.start(done);
+
+      contentMainBlockSelector = '.demo-main';
```

Comment:
this is const

Reply:
do you mean this?

`CONTENT_MAIN_BLOCK_SELECTOR = '.demo-main';`

Reply:
@alectykvach